modules/ztp-site-planning-for-du-deployments.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-site-planning-for-du-deployments_{context}"]
+= Site planning considerations for distributed unit deployments
+
+Site planning for distributed units (DU) deployments is complex. The following is an overview of the tasks that you complete before the DU hosts are brought online in the production environment.
+
+* Develop a network model. The network model depends on various factors such as the size of the area of coverage, number of hosts, projected traffic load, network slicing, DNS, and DHCP requirements.
+* Decide how many DU radio nodes are required to provide sufficient coverage and redundancy for your network.
+* Develop mechanical and electrical specifications for the DU host hardware.
+* Develop a construction plan for individual DU site installations.
+* Submit purchase orders for host hardware and supporting equipment.
```

Comment:
strike "submit purhcase orders" seems obvious and not really relevant.  If there are no servers there is nothing to deploy